src/Oakton/Oakton.csproj

```diff
@@ -19,7 +19,7 @@
     <ItemGroup>
         <PackageReference Include="JasperFx.Core" Version="1.5.1" />
         <PackageReference Include="Spectre.Console" Version="0.47.0" />
-        <PackageReference Include="Microsoft.Extensions.Hosting" Version="[6.0.0,9.0.0)" />
+        <PackageReference Include="Microsoft.Extensions.Hosting" Version="[6.0.0,10.0.0)" />
```

Comment:
What's the point of this range? 

I suspect that the PR submitter just bumped this automatically, so this is not in any way a remark to this PR. But maybe this PR is an opportunity to change it.

All this does in practice is placing an artificial upper limit on valid versions. Such behavior is typically frowned upon. If someone installs just Oakton in a new project, they will get 6.0.0, because that's how NuGet's version resolution works.

Reply:
Nothing was automatic, but after a discussion with Jeremy on Discord it was decided the  
.net 9 PRs would not change anything regarding existing TFM support. As such, since this already used a version range I simply bumped the maximum to allow for 9.x.x

Happy to fix it in this PR, but for posterity it would be better off being a distinct issue + PR.